src/inst/opts/consul.asc

```diff
@@ -1,7 +1,7 @@
 === Custom Consul services
 
-Consul services check is based on hash of hases hiera named : `profiles::consul::server::subservices` and defined
- in file `$ADMIN/puppet-hpc/hieradata/common.yaml`:
+Consul services check is based on hash of hashes hiera named `profiles::consul::server::subservices`
+and defined in file `$ADMIN/puppet-hpc/hieradata/common.yaml`:
```

Comment:
The sentence ends with a column, hence readers believe the following yaml excerpt is the content of this file, but it is not the case.

Reply:
Pls check latest update...